### PR TITLE
Add dependabot-bundler as triggerable action to the project

### PR DESCRIPTION
**Pull request by Skarlso**
### Description

Create a workflow to run dependabot bundler.
I will also update the user docs in eksctl-ci.
Doc update: https://github.com/weaveworks/eksctl-handbook/pull/41

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: .github/workflows/dependabot-bundler.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Dependabot Bundler
+
+on:
+  workflow_dispatch:
```
Comment (nikimanoledaki) [inline .github/workflows/dependabot-bundler.yaml:4]:
```suggestion
on: workflow_dispatch
```

**Review by nikimanoledaki — APPROVED** [review]
LGTM! 👍

File: .github/workflows/dependabot-bundler.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        with:
+          path: |
+            ~/.cache/go-build/
+            ~/go/pkg/mod/
+          key: go-${{ hashFiles('go.sum') }}
+          restore-keys: |
+            go-
+      - name: Install Dependabot Bundler
+        run: |
+          go install github.com/Skarlso/dependabot-bundler@v0.0.7
```
Comment (cPu1) [inline .github/workflows/dependabot-bundler.yaml:26]: Do we want to fork this to the `weaveworks` org?

File: .github/workflows/dependabot-bundler.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name: Dependabot Bundler
+
+on: workflow_dispatch
+
+jobs:
+  bundler:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (cPu1) [inline .github/workflows/dependabot-bundler.yaml:10]: All other workflows are now using v3.
```suggestion
        uses: actions/checkout@v3
```

File: .github/workflows/dependabot-bundler.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18.x
+      - name: Cache go-build and mod
+        uses: actions/cache@v2
```
Comment (cPu1) [inline .github/workflows/dependabot-bundler.yaml:16]:
```suggestion
        uses: actions/cache@v3
```

File: .github/workflows/dependabot-bundler.yaml (hunk 2 above)
Comment (Skarlso) [reply .github/workflows/dependabot-bundler.yaml:26]: We can. I don't know the consensus on that.

**Review by cPu1 — APPROVED** [review]

**Pull request merged**